orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -67,6 +69,14 @@ Select **Start** and Remote Management window will appear soon:` +
 	"After you start, this window will popup every 15-20 minutes until you finish.",
 ))
 
+var mdmManualMigrationTemplate = template.Must(template.New("mdmManualMigrationTemplate").Parse(`
```

Comment:
nit: if you're not planning to use named templates it's best to leave the name empty.

Reply:
TIL! Good to know.